src/network/networkd-address.c

```diff
@@ -2061,7 +2061,6 @@ static int config_parse_broadcast(
 
         address->broadcast = u.in;
         address->set_broadcast = true;
-        address->family = AF_INET;
```

Comment:
Please update the comment above ("Here, we do not check or set address->family.")